Truck.js

```diff
@@ -1,6 +1,35 @@
 var Car = require('./Car.js');
-
 //Implement your Truck here
-function Truck() {};
+function Truck(make, model, year, color, seats, capacity, cargo) {
+	Car.call(this, make, model, year, color, seats);
+	this.capacity = capacity || 0;
+	this.weight = 0;
+	this.cargo = this.weight;
+
+	};
+// load(pounds) should take a weight in pounds and increase the amount of cargo the truck is carrying IF loading the additional weight won't put it over capacity. Should console.log "loaded <pounds>lbs of cargo." and return true if successful and false if the cargo couldn't be loaded (over capacity)
+var load = function(pounds) {
+	if (pounds + weight < capacity) {
```

Comment:
Since we're adding this method to our prototype, we'll need to write this a little bit differently:
```
Truck.prototype.load = function(amount) {
	if (amount + this.cargo <= this.capacity) {
		this.cargo += amount;
		return true;
	}
	return false;
}
```